.github/workflows/zip-release.yml

```diff
@@ -0,0 +1,26 @@
+
+name: Publish TriReg2 API as zip file to GitHub Artifacts
```

Comment:
nitpick: I am not aware of how experienced you are with github actions, so sorry if over explaining. But in practice long names takes longer to read, clutter the views, and more practically; this name will be displayed within the left `workflow` pane under the Actions/Checks tab. And I imagine it will be either cropped, or line breaked. I've attached an illustration from another project. 

If I may suggest something shorter like: `Publish Artifacts`, `Publish Artifacts (.zip)`, `Release: Publish zip`, etc.  

<img width="583" alt="image" src="https://github.com/trifork/trireg2-api/assets/71636857/639a10e8-de53-4a3b-959d-c8cf598f3d49">

Reply:
Thanks! You are absolutely right about that. I will use a shorter name for the action.